ci/steps.py

```diff
@@ -45,7 +46,8 @@ def create_patch(remote_branch: str):
     info = info.replace('\n', ', ')
     patch = git.diff(f'origin/{remote_branch}')
     patch += '\n'
-    print(f'Creating patch against: {remote_branch}, contains files: {info} and has length: {len(patch)}')
+    print(f'Creating patch against: {remote_branch}, contains files: {info} and has length: \
+        {len(patch)}')
```

Comment:
consider using the continuation properties of brackets, e.g.:
```suggestion
    print(
        f'Creating patch against: {remote_branch}, contains files: {info} and has '
        f'length: {len(patch)}'
    )
```

Reply:
Not sure if this is better.... requires 4 lines instead of 2